docs/Debugging/iOS-Debugging.md

```diff
@@ -1,3 +1,188 @@
+## 1. Using the Debugger
+
+## 1. 디버거 사용하기
+
+After you click the Run button in the workspace toolbar and your app builds successfully, Xcode runs your app and starts a debugging session. You can debug your app directly within the source editor with graphical tools such as data tips and Quick Look for the value of variables.
+
+워크스페이스의 툴바에서 실행 버튼을 누르면 앱은 성공적으로 빌드될 것입니다. Xcode는 앱을 실행시키고 디버깅 세션을 시작합니다. `data tips` 같은 그래픽 툴이나 [Quick Look](<https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/debugging_with_xcode/chapters/quick_look_data_types.html>)을 제공하는 소스 에디터에서 앱을 직접 디버깅할 수 있습니다. 
+
+The debug area and the debug navigator let you inspect the current state of your running app and control its execution.
+
+디버그 영역과 탐색기는 당신이 실행하고 있는 앱의 현재 상태를 검사하도록 하고 그것의 실행을 통제하도록 합니다. 
+
+![](~/docs/Debugging/Images/XC_O_DebugFeatures_2x.png)
```

Comment:
If you click this button, you can check whether your images are visible or not. You should change the paths of image sources to `./docs/...`.

Reply:
![스크린샷 2019-09-18 오후 10 19 38](https://user-images.githubusercontent.com/24688568/65152373-c8154c80-da62-11e9-9864-97797e38e0ce.png)